### DISCUSSION TITLE
Browse the same file *. alog ,Celsius changes the temperature in Fahrenheit,

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1093] [General]
![f836d1ca01cda40ac831ea07beebaf7](https://user-images.githubusercontent.com/116416332/216847963-db47dd92-8a7b-427c-bd2e-4fed5147ae26.png)
[23-01-29_2056意式芝士挞.alog.txt](https://github.com/artisan-roaster-scope/artisan/files/10613907/23-01-29_2056.alog.txt)

**Comment by MAKOMO** [comment]
Profiles are shown according to the selected temperature unit of the Artisan app (under menu Config >> Temperature) independent of the temperature unit used to record them. What is exactly the problem or question!?

**Reply by Terracotta-62** [reply]
No problem, we need the same configuration, thank you!

**Comment by Terracotta-62** [comment]
Please close low-level problems。

**Reply by MAKOMO** [reply]
I don't understand. What problems you refer? Do you experience a bug or not? As said, the temperature unit used to render profiles depend on the temperature unit selected in Artisan and is not based on the profile. Artisan automatically converts between C and F according to the choice of the user.

**Reply by Terracotta-62** [reply]
The view screen cannot automatically switch the true temperature unit according to *.alog, manually configure or load *.aset.
